morpheus/app/worker.py

```diff
@@ -529,7 +529,10 @@ async def _messagebird_get_number_cost(self, number: Number):
             return await self._messagebird_get_mcc_cost(redis, mcc)
 
     async def _messagebird_send_sms(self, sms_data: SmsData):
-        msg_cost = await self._messagebird_get_number_cost(sms_data.number)
+        try:
+            msg_cost = await self._messagebird_get_number_cost(sms_data.number)
+        except AssertionError:
```

Comment:
In general, better to create your own error and then catch it.

so in `_messagebird_get_mcc_cost`:

```py
            async with self.ctx['session'].get(url) as r:
                if not assert r.status == 200:
                    raise MessageBirdExternalError((r.status, await r.text()))
                data = await r.json()
```

Then catch it in `_messagebird_send_sms`. Much clearer, an AssertionError is especially vague and harder to pin down.